usvm-python/usvm-python-main/src/main/java/org/usvm/interpreter/CPythonAdapter.java

```diff
@@ -76,7 +76,8 @@ public class CPythonAdapter {
     public native String getNameOfPythonType(long type);
     public static native int getInstructionFromFrame(long frameRef);
     public static native long getCodeFromFrame(long frameRef);
-    public native long allocateVirtualObject(VirtualPythonObject object);
+    public native long allocateRawVirtualObject(VirtualPythonObject object);
+    public native long allocateVirtualObject(VirtualPythonObject object, byte[] mask);
```

Comment:
Could you please add some kind of mapping from type slot to a bit number that corresponds to it?

For example, you can use an approach similar to [`SymbolicMethodId`](https://github.com/UnitTestBot/usvm/blob/main/usvm-python/usvm-python-annotations/src/main/kotlin/org/usvm/annotations/ids/SymbolicMethodId.kt):

1. Introduce enum values for type slots. They should contain the name of the slot id (like `Py_nb_add`). This value will be used in generated C code. Also, add a field for required bit number. This value should be set from C code.
2. Generate code that sets the bit numbers. [Here is the code](https://github.com/UnitTestBot/usvm/blob/main/usvm-python/usvm-python-annotations/src/main/kotlin/org/usvm/annotations/codegeneration/SymbolicMethodGeneration.kt#L37) where `cRef`s of `SymbolicMethodId` are set. That generated macro is inserted into generated header `SymolicMethods.h` [here](https://github.com/UnitTestBot/usvm/blob/37050f9502fdbbf2f4de66b447086c0bc5a6a641/usvm-python/usvm-python-annotations/src/main/kotlin/org/usvm/annotations/SymbolicMethodProcessor.kt#L31). You can just add your macro there as well.
3. Finally, the generated macro can be called during python initialization (like [here](https://github.com/UnitTestBot/usvm/blob/37050f9502fdbbf2f4de66b447086c0bc5a6a641/usvm-python/cpythonadapter/src/main/c/org_usvm_interpreter_CPythonAdapter.c#L84)).
4. Each [TypeProtocol](https://github.com/UnitTestBot/usvm/blob/37050f9502fdbbf2f4de66b447086c0bc5a6a641/usvm-python/usvm-python-main/src/main/kotlin/org/usvm/machine/types/VirtualTypes.kt#L43) should then refer to one of the type slot id's so that later this could be used to generate needed masks.